easybuild/framework/easyconfig/format/yeb.py

```diff
@@ -141,3 +141,18 @@ def is_yeb_format(filename, rawcontent):
             if line.startswith('name: '):
                 isyeb = True
     return isyeb
+
+def quote_yaml_special_chars(val):
+    """
+    Single-quote values that contain special characters, specifically to be used in YAML context (.yeb files)
+    """
+    if isinstance(val, basestring):
+        if "'" in val:
+            if '"' in val:
+                val = val.replace('"', '\\"')
+            val = '"%s"' % val
+
+        elif YAML_SPECIAL_CHARS.intersection(val):
+            val = "'%s'" % val
```

Comment:
Wait, but there was a difference between single-quoting and double-quoted strings in YAML, right?

Include a comment and reference w.r.t. that in this function please...